printer_mods/roboticator24/side-mount_spoolholder/README.md

```diff
@@ -0,0 +1,30 @@
+![in position](./img/inposition.jpg)
+
+# Side-Mount Spoolholder
+
```

Comment:
Also, if this mod is to be merged, you'll need to remove spaces from STL filenames, and `{LHS,RHS} spool holder {front,rear}.stl` (all 4 spool holder STLs) are not in a printable orientation.